src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -25,138 +25,127 @@ const resolveUnreadMessageAriaLabel = unreadMessageCount => {
     : null;
 };
 
+const toggleEnabled = (featureToggles, toggle) =>
```

Comment:
Did you consider using the `useFeatureToggle` from the platform's feature-toggle's code? I think what you have works, but also wonder if there's value in using an existing implementation